google-cloud-spanner-hibernate-performance-testing/src/main/java/com/google/cloud/spanner/hibernate/ClientLibraryOperations.java

```diff
@@ -72,9 +78,19 @@ public ClientLibraryOperations() {
   }
 
   /**
-   * Creates a database.
+   * Resets the test Spanner database.
    */
-  public void createTestDatabase() {
+  public void resetTestDatabase() {
+    try {
+      databaseAdminClient.dropDatabase(INSTANCE_NAME, DATABASE_NAME);
```

Comment:
Is there a way to test if the table exists before attempting to drop it?

Reply:
The only way is to call databaseClient.listDatabases(..) which returns a paginated list and checking if the database is in there. I just decided this way is probably simpler and would only require 1 trip to the database.